package.json

```
@@ -53,32 +41,26 @@
     "vinyl": "^1.2.0",
     "vorpal": "^1.12.0"
   },
-  "bin": {
-    "fractal": "./bin/fractal"
-  },
-  "engines": {
-    "node": ">= 10.0.0"
-  },
-  "scripts": {
-    "pretest": "symlink-dir . node_modules/@frctl/fractal",
-    "test": "mocha --require test/support/env --reporter spec",
-    "coverage": "nyc --reporter=lcov --reporter=text  npm run test",
-    "contributors:add": "all-contributors add",
-    "contributors:generate": "all-contributors generate"
-  },
   "devDependencies": {
     "all-contributors-cli": "^6.14.1",
     "chai": "^4.1.1",
     "chai-as-promised": "^7.1.1",
     "istanbul": "^0.4.4",
+    "lerna": "^3.20.2",
     "mocha": "^5.0.3",
     "mock-fs": "^4.6.0",
     "nyc": "^15.0.1",
     "pre-commit": "^1.1.3",
     "sinon": "^9.0.1",
     "symlink-dir": "^4.0.1"
   },
-  "publishConfig": {
-    "access": "public"
+  "scripts": {
+    "clean": "lerna clean",
+    "setup": "lerna bootstrap --hoist --strict --ci --force-local",
```

```suggestion
    "bootstrap": "lerna bootstrap --hoist --strict --ci --force-local",
```

How about naming this script `bootstrap` to be aligned with lerna’s?